recipes-core/busybox/busybox/defconfig

```diff
@@ -658,7 +658,7 @@ CONFIG_MOUNT=y
 # CONFIG_FEATURE_MOUNT_VERBOSE is not set
 # CONFIG_FEATURE_MOUNT_HELPERS is not set
 # CONFIG_FEATURE_MOUNT_LABEL is not set
-CONFIG_FEATURE_MOUNT_NFS=y
+CONFIG_FEATURE_MOUNT_NFS=n
```

Comment:
```suggestion
# CONFIG_FEATURE_MOUNT_NFS is not set
```

Reply:
The default value for this feature (which doesn't apply for kernels >2.6) is 'no'. So your suggestion seems like a NOOP. Why do you think it should be "not set" rather than explicitly 'no'?